config/airflow.dev.cfg

```diff
@@ -3,6 +3,8 @@
 endpoint_url = http://localstack:4572
 source_bucket = dev-elife-style-content-adapter-incoming
 destination_bucket = dev-elife-style-content-adapter-expanded
+service_name = content-store
+service_url =
```

Comment:
would expect this to be needed shortly, not in this PR however as it was not talking to that service. Can be left in, little harm

Reply:
yes, in an ideal world I would have a container that either mimics the content-store or can receive a put request. For now I'm just mocking the value in tests but the key name still needs to be present in the config otherwise an exception will be raised.